pilot/proxy/envoy/testdata/cds-ssl-context-migration.json.golden

```diff
@@ -0,0 +1,192 @@
+{
+  "clusters": [
+   {
+    "name": "in.1081",
```

Comment:
where is the alternate port for 16080?

Reply:
Actually this golden file is no longer needed. I was implemented this in the way to add extra CDS (and extra LDS) for every migration port, but then change to add LDS only, and change SDS host when the active state flipped.

On the other hand, based on other comments, may be I should change it back to add CDS/SDS/LDS for migration port, and let the routing rules to shift traffic.